data/brands/amenity/fast_food.json

```
@@ -6380,6 +6380,7 @@
         "brand": "Евро Кебаб",
         "brand:en": "Еuro Кеbаb",
         "brand:ru": "Евро Кебаб",
+        "brand:wikidata": "109943635",
```

Don't forget the `Q`  😄 
 
```suggestion
        "brand:wikidata": "Q109943635",
```

---

> 

I'm not a real developer, I'm just pretending)

---

no worries, computers are all pretend anyway